features/frontend/checkout_finalize.feature

```diff
@@ -85,5 +84,4 @@ Feature: Checkout finalization
           And I select the "Dummy" radio button
           And I press "Continue"
          When I click "Place order"
-         Then I should be on the store homepage
-          And I should see "Thank you for your order!"
\ No newline at end of file
+         Then I should see "Thank you"
```

Comment:
We display name right now and it is random in Behat, I will take care of this in separate PR.